src/Miningcore/Payments/PaymentSchemes/PPSPaymentScheme.cs

```diff
@@ -57,18 +78,23 @@ private class Config
 
         #region IPayoutScheme
 
-        public async Task UpdateBalancesAsync(IDbConnection con, IDbTransaction tx, PoolConfig poolConfig,
+        public async Task UpdateBalancesAsync(IDbConnection con, IDbTransaction tx, PoolConfig poolConfig, ClusterConfig clusterConfig,
             IPayoutHandler payoutHandler, Block block, decimal blockReward)
         {
             var payoutConfig = poolConfig.PaymentProcessing.PayoutSchemeConfig;
 
-            // PPLNS window (see https://bitcointalk.org/index.php?topic=39832)
-            var window = payoutConfig?.ToObject<Config>()?.Factor ?? 2.0m;
+            object blockRewardInCache = cache.Get(BLOCK_REWARD);
```

Comment:
Confirmed that the etherscan API is called the first time in a day. Rest of the payout cycles in a day, the blockReward value is retrieved from cache.